etc/testing/deploy-manifests/validate.sh

```diff
@@ -84,15 +89,21 @@ done
 # validation above, as it should accept any valid kubernetes manifest, and
 # would've caught at least one serialization bug that completely broke 'pachctl
 # deploy' in v1.9.8
-if [[ ! "${is_regenerate}" ]]; then
-  DIFF_CMD="${DIFF_CMD:-diff}"
-  if ! "${DIFF_CMD}" "${here}/test" "${here}/golden"; then
-    echo "Deployment manifest has changed." >/dev/stderr
-    echo "If this deliberate, run:" >/dev/stderr
-    echo "  validate.sh --regenerate" >/dev/stderr
-    echo "  git add etc/testing/deploy-manifests/golden" >/dev/stderr
-    echo "  git commit" >/dev/stderr
-    echo "to replace golden deployment manifests with current output" >/dev/stderr
-    exit 1
-  fi
+echo ""
+echo "Diffing 'pachctl deploy' output with known-good golden manifests."
+echo "If this fails but should pass, run:"
```

Comment:
I don't think this is worded the way you intended. Also, it might make sense to only recommend using the make target rather than the shell script (to avoid regenerating against old versions of pachctl).

Reply:
Done :+1: